features/support/formatter.rb

```diff
@@ -1,222 +1,80 @@
 # frozen_string_literal: true
 
-require "fileutils"
-require "colorator"
-require "cucumber/formatter/console"
 require "cucumber/formatter/io"
 
 module Jekyll
   module Cucumber
     class Formatter
```

Comment:
Since there's no test for the formatter, can you share sample output for this so we can see what it looks like? I'm a little worried about truncation causing confusion about what scenario is described in the output so it would be beneficial to see. Thanks!

Reply:
You can see the sample output in our CI workflow logs for this pull request and compare with existing prints for the counterpart in the `master` branch.